File: FrontEnd_Mobile/BeyondLaDecor/src/models/event.model.ts
import { User } from "./user.model";
import { Package } from "./package.model";
import { Location } from "./location.model";
import { EventType } from "./event-type.model";

export interface Event {
    eventId: number,
    eventType : EventType
    client: User,
    planner?: User,
    package: Package
    date: Date,
    locations: Location[],
    cost : number
}

File: BackEnd/BeyondLaDecor.Api/DomainModels/EventModel.cs
using System;
using System.Collections.Generic;

namespace BeyondLaDecor.Beyond.Api.DomainModels
{
    public class EventModel : DecorEntityApiModel
    {
        public int EventId { get; set; }
        public int EventTypeId { get; set; }
        public EventTypeModel EventType { get; set; }
        public DateTime Date { get; set; }
        public PackageModel Package { get; set; }
        public int? PackageId { get; set; }
        public UserModel Client { get; set; }
        public int ClientId { get; set; }
        public ICollection<TaskModel> Tasks { get; set; }
        public ICollection<LocationModel> Locations { get; set; }
    }
}
